features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/APIProduct/APIProductCreateWrapper.jsx

```diff
@@ -65,10 +66,28 @@ const useStyles = makeStyles((theme) => ({
  */
 export default function ApiProductCreateWrapper(props) {
     const { history } = props;
+    const intl = useIntl();
     const [wizardStep, setWizardStep] = useState(0);
     const [apiResources, setApiResources] = useState([]);
     const { settings } = useAppContext();
 
+    const [policies, setPolicies] = useState([]);
+
+    useEffect(() => {
+        API.policies('subscription').then((response) => {
+            const allPolicies = response.body.list;
+            if (allPolicies.length === 0) {
+                Alert.info(intl.formatMessage({
+                    id: 'Apis.Create.APIProduct.APIProductCreateWrapper.error.policies.not.available',
+                    defaultMessage: 'Throttling policies not available. Contact your administrator',
```

Comment:
```suggestion
                    defaultMessage: 'Throttling policies are not available. Please contact your administrator',
```
😃 

@shaniR Hope this is correct